packages/x-components/src/x-modules/related-tags/store/actions/set-url-params-from-the.url.action.ts

```diff
@@ -34,12 +33,12 @@ const createRelatedTags = (relatedTags: string[], query: string): RelatedTag[] =
  * @public
  */
 export const setUrlParamsFromTheUrl: RelatedTagsXStoreModule['actions']['setUrlParamsFromTheUrl'] =
-  ({ commit }, { query, relatedTag }: Dictionary<UrlParamValue>) => {
-    const newRelatedTag = relatedTag as string[];
-    const newQuery = query as string;
+  ({ commit }, { query, relatedTag }: UrlParams) => {
+    const newRelatedTags = relatedTag;
```

Comment:
I think it's `newRelatedTag` as you received the new one.

Reply:
Also this const it's not longer needed, previously it was necessary in order to cast from unknown to string.